docs/examples/infer-orientation-fit-pvwatts.py

```diff
@@ -0,0 +1,88 @@
+"""
+Infer System Orientation (Azimuth & Tilt) using PVWatts
+=======================================================
+
+Infer the azimuth and tilt of a system using PVWatts data
+"""
+
+# %%
+# Identifing and/or validating  the azimuth and tilt information for a
+# system is important, as these values must be correct for future degradation
+# and system yield analysis. This example shows how to use
+# :py:func:`pvanalytics.system.infer_orientation_daily_peak` to estimate
+# a system's azimuth and tilt, using the system's known latitude-longitude
+# coordinates and an associated AC power time series.
+
+import pvanalytics
+from pvanalytics import system as sys
```

Comment:
Same comment re: aliasing to standard lib module names

Reply:
Changed to 'from pvanalytics import system ' in commit from pvanalytics import system